docs/dsp-tools-create.md

```diff
@@ -250,194 +251,201 @@ A resource can be assigned to a taxonomic node within its properties. So a resou
-  level and another level), you don't have child nodes. Therefore the nodes element can be omitted in case of a flat
+- _comments_: Language dependent comments in the form `{ "<lang>": "<comment>", ... }`. The comments element 
+  is optional.
+- _nodes_: Array of sub-nodes. If you have a non-hierarchical taxonomy (i.e. a taxonomy with only 2 levels, the root 
+  level and another level), you don't have child nodes. Therefore, the nodes element can be omitted in case of a flat
   taxonomy.
-  
-Each list must have exactely one root node which has the same form bu denotes the list itself.
 
 Here is an example on how to build a taxonomic structure with the help of JSON:
 
 ```json
-"lists": [
-  {
-    "name": "my_list",
-    "labels": {"en": "Disciplines of the Humanities"},
-    "comments": {"en": "This ist is just a silly example", "fr": "un example un peu fou"},
-    "nodes": [
-      {
-        "name": "node_1_1",
-        "labels": {"en": "Performing arts"},
-        "comments": {"en": "Arts that are events", "de": "Künste mit performativem Character"},
-        "nodes": [
-            {
-            "name": "node_2_2",
-            "labels": {"en": "Music"},
-            "nodes": [
-              {
-                "name": "node_3_3",
-                "labels": {"en": "Chamber music"}
-              },
-              {
-                "name": "node_4_3",
-                "labels": {"en": "Church music"}
-              },
-              {
-                "name": "node_5_3",
-                "labels": {"en": "Conducting"},
-                "nodes": [
-                  {
-                    "name": "node_6_4",
-                    "labels": {"en": "Choirs"}
-                  },
-                  {
-                    "name": "node_7_4",
-                    "labels": {"en": "Orchestras"}
-                  }
-                ]
-              },
-              {
-                "name": "node_8_3",
-                "labels": { "en": "Music history" }
-              },
-              {
-                "name": "node_9_3",
-                "labels": {"en": "Musictheory"}
-              },
-              {
-                "name": "node_10_3",
-                "labels": {"en": "Musicology"
-              },
-              {
-                "name": "node_11_3",
-                "labels": {"en": "Jazz"}
-              },
+{
+  "lists": [
+    {
+      "name": "my_list",
+      "labels": {"en": "Disciplines of the Humanities"},
+      "comments": {"en": "This ist is just a silly example", "fr": "un example un peu fou"},
+      "nodes": [
+        {
+          "name": "node_1_1",
+          "labels": {"en": "Performing arts"},
+          "comments": {"en": "Arts that are events", "de": "Künste mit performativem Character"},
+          "nodes": [
               {
-                "name": "node_12_3",
-                "labels": {"en": "Pop/Rock/Blues"}
-              }
-            ]
-          }
-        ]
-      },
-      {...},{...}
```

Comment:
You systematically removed those `...`s. I'm wondering however if they aren't helpful to users, as they suggest that something needs to go there; and if the user doesn't fill it in, the editor will yell at them. (Counts for all the other instances as well.)

Reply:
I see your point, I changed it back to the version with three dots.